bundle/config/mutator/run_as.go

```diff
@@ -53,14 +53,14 @@ func (e errBothSpAndUserSpecified) Error() string {
 }
 
 func validateRunAs(b *bundle.Bundle) error {
-	runAs := b.Config.RunAs
-
-	// Error if neither service_principal_name nor user_name are specified
-	if runAs.ServicePrincipalName == "" && runAs.UserName == "" {
+	// Error if neither service_principal_name nor user_name are specified, but the
+	// run_as section is present.
+	if b.Config.Value().Get("run_as").Kind() == dyn.KindNil {
```

Comment:
This should also hit if it is specified but an empty map.

Reply:
Thanks! Added the error for that case, as well as coverage for all 4 permutations of this.